src/System.Transactions/ref/System.Transactions.cs

```diff
@@ -0,0 +1,264 @@
+        void Prepare(System.Transactions.PreparingEnlistment preparingEnlistment);
+        void Rollback(System.Transactions.Enlistment enlistment);
+    }
+    public partial interface IPromotableSinglePhaseNotification : System.Transactions.ITransactionPromoter
+    {
+        void Initialize();
+        void Rollback(System.Transactions.SinglePhaseEnlistment singlePhaseEnlistment);
+        void SinglePhaseCommit(System.Transactions.SinglePhaseEnlistment singlePhaseEnlistment);
+    }
+    public partial interface ISimpleTransactionSuperior : System.Transactions.ITransactionPromoter
+    {
+        void Rollback();
+    }
+    public partial interface ISinglePhaseNotification : System.Transactions.IEnlistmentNotification
+    {
+        void SinglePhaseCommit(System.Transactions.SinglePhaseEnlistment singlePhaseEnlistment);
+    }
+    public enum IsolationLevel
+    {
+        Chaos = 5,
+        ReadCommitted = 2,
+        ReadUncommitted = 3,
+        RepeatableRead = 1,
+        Serializable = 0,
+        Snapshot = 4,
+        Unspecified = 6,
+    }
+    public partial interface ITransactionPromoter
+    {
+        byte[] Promote();
+    }
+    public partial class PreparingEnlistment : System.Transactions.Enlistment
+    {
+        internal PreparingEnlistment() { }
+        public void ForceRollback() { }
+        public void ForceRollback(System.Exception e) { }
+        public void Prepared() { }
+        public byte[] RecoveryInformation() { return default(byte[]); }
+    }
+    public partial class SinglePhaseEnlistment : System.Transactions.Enlistment
+    {
+        internal SinglePhaseEnlistment() { }
+        public void Aborted() { }
+        public void Aborted(System.Exception e) { }
+        public void Committed() { }
+        public void InDoubt() { }
+        public void InDoubt(System.Exception e) { }
+    }
+    public sealed partial class SubordinateTransaction : System.Transactions.Transaction
+    {
+        public SubordinateTransaction(System.Transactions.IsolationLevel isoLevel, System.Transactions.ISimpleTransactionSuperior superior) { }
+    }
+    public partial class Transaction : System.IDisposable, System.Runtime.Serialization.ISerializable
+    {
+        internal Transaction() { }
+        public static System.Transactions.Transaction Current { get { return default(System.Transactions.Transaction); } set { } }
+        public System.Transactions.IsolationLevel IsolationLevel { get { return default(System.Transactions.IsolationLevel); } }
+        public System.Guid PromoterType { get; }
+        public System.Transactions.TransactionInformation TransactionInformation { get { return default(System.Transactions.TransactionInformation); } }
+        public event System.Transactions.TransactionCompletedEventHandler TransactionCompleted { add { } remove { } }
+        //protected System.IAsyncResult BeginCommitInternal(System.AsyncCallback callback) { return default(System.IAsyncResult); }
```

Comment:
>protected System.IAsyncResult BeginCommitInternal [](start = 10, length = 49)

Is the standard for .NET Core to stick with Task or should we continue to use Begin/End?